resources/carrier/en/234.txt

```diff
@@ -151,7 +155,13 @@
 2347028|Starcomms
 2347029|Starcomms
 234703|MTN
-234704|Visafone
+2347040|Visafone
```

Comment:
@ifiok,
True. However the link of NCC website in ITU is broken. We found that in few cases NCC webpage is incomplete. Ex: 707 are mentioned as "For Withdrawal", we found working numbers at: https://www.naijatechguide.com/2008/10/reltel-rebrands-to-zoom-mobile-launches.html 

So we do support ranges case by case.